README.md

```diff
@@ -3,40 +3,51 @@
 Follow the instructions below to set up a Flask application in your environment:
 
 ### 1. Check Pip Version:
+
 Before you start, ensure you have `pip` installed. Check its version with:
+
 ```bash
 pip --version
 ```
 
 ### 2. Upgrade Pip, Setuptools, and Virtualenv:
+
 It's a good practice to keep your tools updated. Run the following command:
+
 ```bash
 python -m pip install --upgrade pip setuptools virtualenv
```

Comment:
we should reconsider if we want to make these recommendations - or at least specify they are not needed when using a devcontainer as the environment is already isolated to the devcontainer!

Reply:
I suggest we fix this in the issue to add devcontainers configuration #32

Reply:
not sure how it relates, this is a documentation update and that #32 is just a single command change.

But you can create a separate issue and have @CFIALeronB do it

Reply:
That issue actually adds devcontainer to the project, so virtualenvs are no longer needed.